### ISSUE TITLE
Having trouble installing / No longer works

### ISSUE THREAD

**Issue by AngloYorkshire** [madmicio/LG-WebOS-Remote-Control#98]
Been trying to install this over the past hour or so and done both via HACS and manually on my Virtual Box version.

I add the resources to my main configuration.yaml file as no longer add resources the way it says in HA (I'm on July 2022 version)

But anyway I try it cannot see the cards

Many thanks

**Comment by randoguyname** [comment]
Have you tried going through the manual card and inputting the following (or similar) configuration:
```yaml
type: 'custom:lg-remote-control'
entity: media_player.<your_entity_name>
```
I had this same issue, I'm not sure if this is meant to have an editor yet.

**Comment by AngloYorkshire** [comment]
Yes I have tried it manually and it says the card is not avaialble.
I think the problem is the Resources and the location as the latest versions of HA as removed their location since this was posted , so not 100% sure where to put the rsources.
Many thanks for getting in touch so quickly

**Comment by Clueless2015** [comment]
A similar issue with my 2022.07.0 build.

I see the HACS installed resource in the GUI under Settings\Dashboard\Resources

/hacsfiles/LG-WebOS-Remote-Control/lg-remote-control.js?hacstag=......

When i try a custom card in Lovelace:

"No card type found"

- type: custom:lg-remote-control
  entity: media_player.lg......
  mac: xxxxxxxxxxxxxxxx
  sources:

**Comment by ntindle** [comment]
If anyone runs into this later, removing the dash before type was the change

**Comment by YanisKyr** [comment]
@madmicio might be a good idea to remove the hyphen from this point in the readme :)
https://github.com/madmicio/LG-WebOS-Remote-Control#lovelace-config-default-view
